lib/topology_lib_iperf/library.py

```diff
@@ -67,34 +76,45 @@ def server_start(enode, state, port, interval=1, udp=False):
     if udp is True:
         cmd.append('-u')
 
-    cmd.append('2>&1 > /tmp/iperf_server.log &')
+    cmd.append('2>&1 > /tmp/iperf_server')
+    if instance_id is not None:
+        cmd.append('-{}'.format(instance_id))
+    cmd.append('.log &')
 
-    state.server_pid = parse_pid(enode(' '.join(cmd), shell='bash'))
+    state.server_pids[instance_id] = parse_pid(enode(' '.join(cmd),
+                                                     shell='bash'))
 
```

Comment:
None is a perfectly valid key for a dictionary in Python and my intent was that None is the default key to preserve original functionality as much as possible.
It's something I can change of course to simplify lines of code.